source/example-smart-app/lib/css/cerner-smart-embeddable-lib-1.0.0.min.css

```diff
@@ -0,0 +1 @@
+html[hidden]{display:none}
```

Comment:
We should use this opportunity to update these files to consume the latest version of cerner-smart-embeddable-lib from here : https://github.com/cerner/cerner-smart-embeddable-lib/tree/v1.4.0/dist

Reply:
No problem. For updating this CSS, could we just replace this file with https://github.com/cerner/cerner-smart-embeddable-lib/blob/v1.4.0/dist/css/cerner-smart-embeddable-lib-1.4.0.min.css? The same for js? https://github.com/cerner/cerner-smart-embeddable-lib/blob/v1.4.0/dist/js/cerner-smart-embeddable-lib-1.4.0.min.js